src/meidocument.cpp

```diff
@@ -95,9 +95,8 @@ MeiElement* mei::MeiDocument::getElementById(string id) {
     map<string, MeiElement*>::iterator iter = idMap.find(id);
     if (iter == idMap.end()) {
         return NULL;
-    } else {
-        return iter->second;
     }
+    return iter->second;
```

Comment:
There are opinions that an `if` should always be followed by an `else` to completely avoid any possible ambiguity. This would speak for keeping the code as it is.